main.py

```diff
@@ -75,6 +78,13 @@ def export_streams_to_clipboard(self):
         clipboard.clear(mode=clipboard.Clipboard)
         clipboard.setText(text, mode=clipboard.Clipboard)
 
+    def import_streams_from_clipboard(self):
```

Comment:
Yeah now I am confused myself. Pretty sure we were supposed to have a dedicated button for this. But w/e not that important atm.